pcsx2/gui/MainFrame.cpp

```diff
@@ -48,8 +49,14 @@ wxMenu* MainEmuFrame::MakeStatesSubMenu(int baseid, int loadBackupId) const
 		wxMenuItem* m = mnuSubstates->Append(loadBackupId, _("Backup"));
 		m->Enable(false);
 	}
-
+	//  Implement custom hotkeys (F2) + (Shift + F2) with translatable string intact + not blank in GUI.
+	// baseid in the negatives will order in a different section, so if you want to increase more slots you can still easily do this, as -1 it will have the same function as opening file for savestates, which is bad
+	// For safety i also made them inactive aka grayed out to signify that's it's only for informational purposes, in the future this can still be expanded to cycle savestates in the GUI.
 	mnuSubstates->Append(baseid - 1, _("File..."));
+	wxMenuItem* CycleNext = mnuSubstates->Append(baseid - 2, _("Cycle to next slot") + wxString("  ") + fmt::format("({})", wxGetApp().GlobalAccels->findKeycodeWithCommandId("States_CycleSlotForward").toTitleizedString()));
+	CycleNext->Enable(false);
+	wxMenuItem* CycleBack = mnuSubstates->Append(baseid - 3, _("Cycle to previous slot") + wxString("  ") + fmt::format("({})", wxGetApp().GlobalAccels->findKeycodeWithCommandId("States_CycleSlotBackward").toTitleizedString()));
+	CycleBack->Enable(false);
```

Comment:
yeah it would be nice to have these enable and actual commands, but perhaps a separate PR.
Here's some of the lines you'd be interested in:
https://github.com/PCSX2/pcsx2/blob/7b7278bc85fa3becd39c47168fe6c912523b4d43/pcsx2/gui/UpdateUI.cpp#L50-L51
https://github.com/PCSX2/pcsx2/blob/3c8d4029ddb601063bb011987824a879e29f7124/pcsx2/gui/Saveslots.cpp#L155-L165

Reply:
I'll look into in the future, but i don't want to break anything for now especially with the current scope and size of this PR.